### ISSUE TITLE
Send Email BEFORE new user approval

### ISSUE THREAD

**Issue by Gilbertdelyon** [humhub/humhub#6166]
Normal HH Approval Process:
- Someone fills out the form to request to join the network
- the administrator approves/refuses
- an email is sent accordingly to the applicant

Our Approval Process:
- step 1:  Someone fills out the form to request to join the network
- **step 2**:  the administrator  sends a private Email to the applicant and asks for profile, motivation, and various other items
- step 3: The administrator receives the answer and approves/refuses consequently
- step 4: an email is sent accordingly to the applicant

Here is the Issue:
Step 2 is not possible from the HH site. We have  to use an email client.
We cannot use the native "email" functionnality because the applicant address is (of course) not selectebale.

Request:
It would be nice to be able to send an email directly from the site.
**For example** as follows: ( Click on the button would open a form, prefilled with the applicant email + text area (plain text would be sufficient)
![image](https://user-images.githubusercontent.com/91057837/224841190-4efdd026-e427-4af4-a86c-b34a053695bb.png)

**Another possible (very nice!) way**: 
in admin/users/settings we have 2 text input fields for email content
- Default content of the registration approval email
- Default content of the registration denial email
It would be nice to have a third one:
- Default content of request for application  motivation email
And the content of this field would sent to the applicant when we click on the "orange button"

**Comment by luke-** [comment]
How should replies to this email be handled? Also via HumHub?

 Maybe it could be  also a "light" solution to simply make the email "clickable" via "mailto"?

**Comment by Gilbertdelyon** [comment]
>_" How should replies to this email be handled? Also via HumHub?"_
No, the applicant would answer directly to the site email address from his email client. 
This would may be need a specific amail template without any "see online" link

>_"Maybe it could be also a "light" solution to simply make the email "clickable" via "mailto"?"_
That's a good idea, but, please, what do you have in mind?
- this "mailto" would drive you to a specific site form?
- or to the to the email client of the machine you are working on?
We would prefer the first way, because ou site sends all emails via [SendinBlue](https://de.sendinblue.com/) with many benefits we dont have with a mail client (logs, mail opened, mail clicked, bounces...)

**Comment by luke-** [comment]
Yes, the idea is simply to open the user's local email client. 

Other solutions may add too much complexity (form, handling of replies) for this edge case  and would then rather be something for a separate module.

**Comment by Gilbertdelyon** [comment]
OK, I understand
I will try do make something more simple in a php custom page (in between of present way and the "ideal way")

**Comment by luke-** [comment]
I've added an issue for the "easy" variant here:
https://github.com/humhub/humhub/issues/6171

**Comment by petersphilo** [comment]
Why don't you just add your questions from your step 2/3 directly to the initial HumHub registration form, and make them required fields?
Seems to me like you're just looking for additional information to decide whether to approve/reject..
Just add your fields to the Reg. form in HumHub..

**Comment by Gilbertdelyon** [comment]
So,  I added a new textarea so that applicant can now write his motivations.
This was my first experience in this matter and I have a few comments:
- It's not possible to add a placeholder in the field
- The field is too narrow and not high enough, so that the applicant doesn't have enough room to write his text. Not very convenient.
- The legend does not accept html, but only  raw text. `crlf` are not converted .

So I added a few lines of javascript in our custom theme, and here it is.

**In Native theme**:
![image](https://user-images.githubusercontent.com/91057837/226011338-172e0e63-b615-4837-8a96-f258dcdbba5e.png)

**With a few lines of js in our custom theme**
![image](https://user-images.githubusercontent.com/91057837/226012059-595fe747-3310-4193-8b33-990039c93aa0.png)

So far so good !
Thanks again to @petersphilo
